webapp/channels/src/sass/responsive/_mobile.scss

```diff
@@ -2299,3 +2299,11 @@ label {
         font-size: 16px;
     }
 }
+
+@media screen and (max-width: 540px) {
+    .modal-body {
+        >input {
+            margin: 2px 2px 22px 0;
+        }
```

Comment:
i am a bit hesitant here with this wide reach selector which could effect other places as well, can we scope down this further?